pages/common/kdeconnect-cli.md

```diff
@@ -0,0 +1,35 @@
+# kdeconnect-cli
+
+> KDE Connect CLI.
+
+- List all devices:
+
+`kdeconnect-cli --list-devices`
+
+- List all reachable devices:
+
+`kdeconnect-cli --list-available`
+
+- Request pairing with a specific device, specifying its ID:
+
+`kdeconnect-cli --pair --device {{device_id}}`
+
+- Request pairing with a specific device, specifying its name:
+
+`kdeconnect-cli --pair --name {{device_name}}`
+
+- Ring a specific device:
+
+`kdeconnect-cli --ring --name {{device_name}}`
+
+- Send an SMS with an optional attachment to a specific number:
+
+`kdeconnect-cli --name {{device_name}} --send-sms {{message}} --destination {{phone_number}} --attachment {{path/to/file}}`
+
+- Unlock a specific device:
+
+`kdeconnect-cli --name {{device_name}} --unlock`
+
+- Send a key to a specific device:
```

Comment:
What kind of key is this referring to?

Reply:
A key press

Reply:
I tried it, but I don't understand how this command works.

Reply:
It should work that way. However it doesn't work for me either, because a lot of KDE Connect is pretty broken.

Reply:
Hmm, well maybe we shouldn't even document it then 🤔

Reply:
I think it's better to have it documented for the cases it does work, than not having it documented because of the cases it doesn't work.

Reply:
How about
```suggestion
- Simulate a key presses on a specific device:
```

Reply:
Go for it 👍🏻